### PR TITLE
Add and Remove

### PR DESCRIPTION
**Pull request by digia3d**
In this part of the project I add new js files with functions for adding, deleting, and editing task descriptions.

### CODE REVIEW

File: src/modules/displayData.js [hunk 1]
```diff
@@ -1,26 +1,24 @@
...
+    completed: true,
+  },
+  {
+    index: 2,
+    description: 'Clean house',
+    completed: true,
+  },
+  {
+    index: 3,
+    description: 'Wash dishes',
```
Comment (SajjadAhmad14) [inline src/modules/displayData.js]: Please remove all hardcoded items from the tasks array.

File: src/modules/addTask.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+const addTask = (tasklist, newTask) => {
```
Comment (SajjadAhmad14) [inline src/modules/addTask.js]: The app should allow the user to add, remove, and edit the tasks. The user is not able to perform these operations. Kindly go through the project requirements again and make sure the app is working as expected.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Alex, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
Git flow is followed. :heavy_check_mark:
README is professional. :heavy_check_mark:
## To Improve 🙏

- [x] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/modules/addTask.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    addTask(description) {
+      const task = {
+        description,
+        completed: false,
+        index: this.listArray.length + 1,
+        edit: false,
+      };
+      this.listArray = [...this.listArray, task];
+      this.updateTasks();
+    }
```
Comment (elisha2kyakpo1) [inline src/modules/addTask.js:34]: - [ ] Kindly fix the "add" functionality, I couldn't add a task(todo) in the local storage. Let me know if there is anything I'm missing. 

todo

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @digia3d, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- You have made great progress 👏

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/addTask.js (hunk 3 above)
Comment (digia3d) [reply src/modules/addTask.js:34]: @elisha2kyakpo1 Elisha Kyakopo Hi, I don't understand why are you asking me to put functionality, when that task is for tomorrow? Can you please explain it to me? Thank you.

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi @digia3d,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

thumbs-up-agree


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**